Sources/BraveWallet/Crypto/NFT/NFTDetailView.swift

```diff
@@ -60,129 +78,237 @@ struct NFTDetailView: View {
+                  openWalletURL(url)
+                }
+              } else {
+                if let url = nftDetailStore.networkInfo.splTokenBlockExplorerURL(nftDetailStore.nft) {
+                  openWalletURL(url)
                 }
               }
-            } else {
+            } label: {
               HStack {
-                Text(Strings.Wallet.tokenMintAddress)
-                  .font(.headline.weight(.semibold))
-                Spacer()
-                Button(action: {
-                  if WalletConstants.supportedTestNetworkChainIds.contains(nftDetailStore.networkInfo.chainId) {
-                    if let components = nftDetailStore.networkInfo.blockExplorerUrls.first?.separatedBy("/?cluster="), let baseURL = components.first {
-                      let cluster = components.last ?? ""
-                      if let nftURL = URL(string: "\(baseURL)/address/\(nftDetailStore.nft.contractAddress)/?cluster=\(cluster)") {
-                        openWalletURL(nftURL)
-                      }
-                    }
-                  } else {
-                    if let explorerURL = nftDetailStore.networkInfo.blockExplorerUrls.first, let nftURL = URL(string: "\(explorerURL)/address/\(nftDetailStore.nft.contractAddress)") {
-                      openWalletURL(nftURL)
-                    }
-                  }
-                }) {
-                  HStack {
-                    Text("\(nftDetailStore.nft.contractAddress.truncatedAddress)")
-                    Image(systemName: "arrow.up.forward.square")
-                  }
-                  .foregroundColor(Color(.braveBlurple))
-                }
+                Text(nftDetailStore.nft.contractAddress.truncatedAddress)
+                Image(systemName: "arrow.up.forward.square")
               }
+              .font(.subheadline)
+              .foregroundColor(Color(.braveBlurpleTint))
             }
           }
-          .foregroundColor(Color(.braveLabel))
+          NFTDetailRow(title: Strings.Wallet.nftDetailBlockchain) {
+            Text(nftDetailStore.networkInfo.chainName)
+              .font(.subheadline)
+              .foregroundColor(Color(.braveLabel))
+          }
+          NFTDetailRow(title: Strings.Wallet.nftDetailTokenStandard) {
+            Text(nftDetailStore.nft.isErc721 ? Strings.Wallet.nftDetailERC721 : Strings.Wallet.nftDetailSPL)
+              .font(.subheadline)
+              .foregroundColor(Color(.braveLabel))
+          }
         }
-        if isSVGImage {
-          Text(Strings.Wallet.nftDetailSVGImageDisclaimer)
-            .multilineTextAlignment(.center)
-            .font(.footnote)
-            .foregroundColor(Color(.secondaryBraveLabel))
-            .frame(maxWidth: .infinity)
+        .listRowBackground(Color(.secondaryBraveGroupedBackground))
+      } header: {
+        Text(Strings.Wallet.nftDetailOverview)
+          .listRowInsets(.zero)
```

Comment:
I don't think we want `listRowInsets(.zero)` on any of these headers.
NFT Detail | Settings / Default
--|--
<img width="116" alt="NFT Detail" src="https://github.com/brave/brave-ios/assets/5314553/80913bf7-e6ee-4f9d-b059-9cf9d3524335"> | <img width="148" alt="Settings : default" src="https://github.com/brave/brave-ios/assets/5314553/c77a6b70-b038-45df-839b-3d4bd7f5fb10">